In [1]:
panjang_balok = int(input("Masukkan panjang balok: "))
lebar_balok = int(input("Masukkan lebar balok: "))
tinggi_balok = int(input("Masukkan tinggi balok: "))

# menghitung volume balok
volume_balok = panjang_balok * lebar_balok * tinggi_balok

# menghitung keliling balok
keliling_balok = 4 * (panjang_balok+lebar_balok+tinggi_balok)

# menampilkan hasil
print("volume balok: ", volume_balok)
print("keliling balok: ", keliling_balok)

volume balok:  168
keliling balok:  68


Analisis Kode untuk Menghitung Volume dan Keliling Balok
Fungsi Kode:

Kode ini memungkinkan pengguna untuk memasukkan nilai panjang, lebar, dan tinggi balok, kemudian menghitung dan menampilkan volume dan keliling balok.

Langkah-langkah:
Meminta Input:
Kode menggunakan input() untuk meminta pengguna memasukkan nilai panjang, lebar, dan tinggi balok. Nilai input disimpan dalam variabel panjang_balok, lebar_balok, dan tinggi_balok.

Menghitung Volume Balok:
Rumus volume_balok = panjang_balok * lebar_balok * tinggi_balok digunakan untuk menghitung volume balok. Hasilnya disimpan dalam variabel volume_balok.

Menghitung Keliling Balok:
Rumus keliling_balok = 4 * (panjang_balok + lebar_balok + tinggi_balok) digunakan untuk menghitung keliling balok. Hasilnya disimpan dalam variabel keliling_balok.

Menampilkan Hasil:
Kode menggunakan print() untuk menampilkan nilai volume balok dan keliling balok yang telah dihitung.

hasil dari codingan ini dengan memasukkan nilai panjang,lebar dan tinggi dengan panjang (4) lebar (6) dan tinggi adalah (7)
dan mendapatkan hasil dari dari volume balok denan hasil nya adalah (168) dan keliling balok dengan hasil (68)